src/SprykerEco/Zed/Unzer/Business/Refund/RefundStrategy/ExpensesRefundStrategyResolver/UnzerExpensesRefundStrategyResolver.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Refund\RefundStrategy\ExpensesRefundStrategyResolver;
+
+use SprykerEco\Zed\Unzer\Business\Exception\UnzerException;
+use SprykerEco\Zed\Unzer\Business\Refund\RefundStrategy\UnzerExpensesRefundStrategyInterface;
+
+class UnzerExpensesRefundStrategyResolver implements UnzerExpensesRefundStrategyResolverInterface
+{
+    /**
+     * @var array<int, \Closure>
+     */
+    protected $unzerRefundStrategiesCollection;
```

Comment:
```suggestion
    protected $unzerRefundStrategies;
```
collection usually means something specific and not regular array